### ISSUE TITLE
<input> is not an empty element

### ISSUE THREAD

**Issue by petdance** [petdance/html-tagset#11]
Migrated from [RT](https://rt.cpan.org/Ticket/Display.html?id=57496), originally by Tomas Doran @bobtfish

```
$ perl -MHTML::Tagset -e'warn $HTML::Tagset::emptyElement{input}'
1 at -e line 1.
```

This appears entirely wrong. `<input>` tags can contain content as this is how you set previous values for them..

```
E.g. <input type="text" name="foo">initial value</input>
```

**Comment by petdance** [comment]
Comments from @PhilterPaper on original RT ticket.

I don't think this is correct. `<input>` uses a  value="text"  attribute rather than child content. Is this perhaps a very, very old version of HTML?

Note that there is also a "placeholder" attribute, used for a prompt string (but not a default value). I don't think both "value" and "placeholder" can be used at the same time (since they both put stuff in the input box), but I don't see any restrictions on this.

**Comment by haarg** [comment]
This is incorrect. As shown in [the spec](https://html.spec.whatwg.org/multipage/input.html#the-input-element), input tags have no ending tag, thus they cannot have any content. To provide an initial value for an input, you use the `value` attribute.

**Comment by oalders** [comment]
I think we can close this issue.
